frontend/public/components/monitoring/dashboards/error.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { Alert } from '@patternfly/react-core';
+
+const ErrorAlert: React.FC<Props> = ({ message, title = 'An error occurred' }) => (
+  <Alert isInline className="co-alert" title={title} variant="danger">
```

Comment:
If the message might get long, we should add `--scrollable`

```suggestion
  <Alert isInline className="co-alert co-alert--scrollable" title={title} variant="danger">
```

Reply:
👍